scripts/test/generate_lld_tests.py

```diff
@@ -57,13 +57,12 @@ def generate_wat_files(llvm_bin, emscripten_root):
             '-nostdinc',
             '-Xclang', '-nobuiltininc',
             '-Xclang', '-nostdsysteminc',
-            '-Xclang', '-I%s/system/include' % emscripten_root,
+            '-Xclang', '-I%s/include' % emscripten_sysroot,
             '-O1',
         ]
 
         link_cmd = [
             os.path.join(llvm_bin, 'wasm-ld'), '-flavor', 'wasm',
-            '-z', '-stack-size=1048576',
```

Comment:
There was a typo on this line. The leading `-` meant that the linker was ignoring this option, but newer versions of wasm-ld fail with an error.    

I'm not sure why this was added, but removing it keeps the test expectations unchanged in their stack size.  Fixing the option by removing the leading `-` will cause all the tests to have larger stack sizes..